spec/graphql/anycable_spec.rb

```diff
@@ -112,37 +112,76 @@
-    end
+    context "with default config.redis-prefix" do
+      before do
+        GraphQL::AnyCable.config.use_client_provided_uniq_id = false
+      end
 
-    before do
-      AnycableSchema.execute(
-        query: query,
-        context: { channel: channel, subscription_id: subscription_id },
-        variables: {},
-        operation_name: "SomeSubscription",
-      )
-    end
+      before do
+        AnycableSchema.execute(
+          query: query,
+          context: { channel: channel, subscription_id: subscription_id },
+          variables: {},
+          operation_name: "SomeSubscription",
+          )
+      end
 
-    after do
-      GraphQL::AnyCable.config.use_client_provided_uniq_id = false
-    end
+      after do
+        GraphQL::AnyCable.config.use_client_provided_uniq_id = false
+      end
 
-    let(:redis) { AnycableSchema.subscriptions.redis }
+      let(:redis) { AnycableSchema.subscriptions.redis }
 
-    subject do
-      AnycableSchema.subscriptions.delete_channel_subscriptions(channel)
+      subject do
+        AnycableSchema.subscriptions.delete_channel_subscriptions(channel)
+      end
+
+      it "removes subscription from redis" do
+        expect(redis.exists?("graphql-subscription:some-truly-random-number")).to be true
+        expect(redis.exists?("graphql-channel:some-truly-random-number")).to be true
+        expect(redis.exists?("graphql-fingerprints::productUpdated:")).to be true
+        subject
+        expect(redis.exists?("graphql-channel:some-truly-random-number")).to be false
+        expect(redis.exists?("graphql-fingerprints::productUpdated:")).to be false
+        expect(redis.exists?("graphql-subscription:some-truly-random-number")).to be false
+      end
     end
 
-    it "removes subscription from redis" do
-      expect(redis.exists?("graphql-subscription:some-truly-random-number")).to be true
-      expect(redis.exists?("graphql-channel:some-truly-random-number")).to be true
-      expect(redis.exists?("graphql-fingerprints::productUpdated:")).to be true
-      subject
-      expect(redis.exists?("graphql-channel:some-truly-random-number")).to be false
-      expect(redis.exists?("graphql-fingerprints::productUpdated:")).to be false
-      expect(redis.exists?("graphql-subscription:some-truly-random-number")).to be false
+    context "with different config.redis-prefix" do
+      before do
+        GraphQL::AnyCable.config.use_client_provided_uniq_id = false
+        GraphQL::AnyCable.config.redis_prefix = "graphql-test"
+      end
+
+      before do
+        AnycableSchema.execute(
+          query: query,
+          context: { channel: channel, subscription_id: subscription_id },
+          variables: {},
+          operation_name: "SomeSubscription",
+          )
+      end
+
+      after do
+        GraphQL::AnyCable.config.use_client_provided_uniq_id = false
+        GraphQL::AnyCable.config.redis_prefix = "graphql"
+      end
```

Comment:
It's better to use `around` and preserve the prev values. See [example](https://github.com/palkan/anyway_config/blob/fe818683abe2444a22015eaa5f2af2e3ff8b8de3/spec/settings_spec.rb#L7C1-L11).

Reply:
thanks!
done!